ACLWeb.js

```diff
@@ -153,13 +202,19 @@ function callTranslator(bibFileURL, type, attachments) {
 			item.repository = "Association for Computational Linguistics"
 			item.complete();
 		});
+		Zotero.debug("translating");
 		translator.translate();
 
 	});
 }
 
 function doWeb(doc, url) {
+
 	var searchResult = true;
+	if (url.lastIndexOf(".bib")>-1 || url.lastIndexOf(".pdf") > -1 || url.lastIndexOf(".") == -1) {
```

Comment:
What's the idea behind `url.lastIndexOf(".") == -1`? That's never going to be true, since there's always a period in the URL (i.e. aclweb.org).